src/Internal/Service/PaymentProcessingService.php

```diff
@@ -127,13 +127,13 @@ public function get_authentication_redirect_url( $intent, int $order_id ) {
 	 * Instantiates a new empty payment context.
 	 *
 	 * @param int  $order_id ID of the order that the context belongs to.
-	 * @param bool $manual_capture Whether manual capture is enabled.
+	 * @param bool $automatic_capture Whether automatic capture is enabled.
```

Comment:
Could you please restore the spacing to align descriptions? It was previously removed in https://github.com/Automattic/woocommerce-payments/commit/dd30c931197f31246f08f258f11bc1d96b445403#diff-6dcdd46e1ba4439fc971a031b5142202702cf501bbc2f2f18892c3d057642930L77.

```suggestion
	 * @param int  $order_id          ID of the order that the context belongs to.
	 * @param bool $automatic_capture Whether automatic capture is enabled.
```

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/7645/commits/e1b3e4ec9400eddd035defbf828dd0e171fc34f4